person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  altr_accessor :name, :age
+  altr_reader :id
```

Comment:
- [ ] You are almost there, everything is looking good! However, there's a typo here, it's actually `attr_accessor attr_reader` which comes from 'attribute. It would be great if you fixed it.